interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer:It is a way to call back an item in a object.
 
-  Researched answer:
+  Researched answer: The .this is a call back function to an object and it allows it to be called muiltple times.
```

Comment:
"this" is a key word that refers to the instance of a class that the key word is found in.